src/Microsoft.Framework.Logging/LoggerFactory.cs

```diff
@@ -15,6 +15,7 @@ public class LoggerFactory : ILoggerFactory
         private readonly Dictionary<string, Logger> _loggers = new Dictionary<string, Logger>(StringComparer.Ordinal);
         private ILoggerProvider[] _providers = new ILoggerProvider[0];
         private readonly object _sync = new object();
+        private LogLevel? _minimumLevel;
```

Comment:
Do we need this? Can't we just set the default?

```C#
public LogLevel MinimumLevel { get; set; } = LogLevel.Verbose;
```

Also the default level is Debug right?